src/components/Message/Message.tsx

```diff
@@ -417,14 +470,22 @@ const MessageWithContext = <
     >;
     if (error) {
       showMessageOverlay(false, true);
-    } else if (goToMessage && quotedMessage) {
-      pressActive.value = false;
-      cancelAnimation(scale);
-      scale.value = withTiming(1, { duration: 100 });
-      goToMessage(quotedMessage.id);
+    } else if (quotedMessage) {
+      onPressQuotedMessage(quotedMessage);
     }
   };
 
+  const onPressQuotedMessage = (
```

Comment:
Should move up so it's created before called just for logical reading.

Reply:
Done!!